packages/w3p/examples/multiple-providers.ts

```diff
@@ -103,6 +104,21 @@ export const useWeb3ProvidersStore = defineStore(STORE_NAME, () => {
        *     ) as RawProvider,
        *   })
        * }
+
+       * Because WalletConnect does not provide an interface for
+       * interacting with Ethereum through `window.ethereum`,
+       * we need to add information about this provider using following
+       * method
```

Comment:
Since WalletConnect lacks an injected provider for interacting with the chain, it's required to add WalletConnect provider configuration with the following method:

Reply:
(WalletConnect can interact not only with the EVM chains JFYI)

Reply:
done